cloudcannon.config.yaml

```diff
@@ -158,4 +158,31 @@ _editables:
     undo: false
     redo: false
     removeformat: false
-    copyformatting: false
\ No newline at end of file
+    copyformatting: false
+    snippet: true
```

Comment:
this adds support for shortcakes in the editor. it also adds the handful of prebuilt shortcodes from CC. You should toggle this new menu to see if you want to offer these shortcodes